code/Control/UserDefinedFormController.php

```diff
@@ -401,7 +402,11 @@ public function process($data, $form)
                     $body = strip_tags($recipient->getEmailBodyContent()) . "\n";
                     if (isset($emailData['Fields']) && !$emailData['HideFormData']) {
                         foreach ($emailData['Fields'] as $field) {
-                            $body .= $field->Title . ': ' . $field->Value . " \n";
+                            if (in_array(SubmittedFileField::class, $field->getClassAncestry())) {
```

Comment:
isn't an `instanceof` check better here?

`if ($field instanceof SubmittedFileField) {`

Reply:
Sure thing @dhensby. I have changed this a requested.